api-client/src/utils/GeoJson.ts

```diff
@@ -0,0 +1,22 @@
+import { Feature, FeatureCollection } from 'geojson'
+import LocationModel from '../models/LocationModel'
+
+export const convertToPoint = (location: LocationModel): Feature => {
+  return {
+    type: 'Feature',
+    geometry: {
+      type: 'Point',
+      coordinates: [Number(location.longitude), Number(location.latitude)]
```

Comment:
`longitude` and `latitude` can be null. This means this function actually should return `Feature | null` if one of lat/long is null.